cmake/Platform/Mac/InstallUtils_mac.cmake.in

```diff
@@ -137,20 +137,20 @@ function(ly_copy source_file target_directory)
 
     endif()
     get_filename_component(target_filename "${source_file}" NAME)
-    file(COPY "${source_file}" DESTINATION "${target_directory}" FILE_PERMISSIONS @LY_COPY_PERMISSIONS@ FOLLOW_SYMLINK_CHAIN)
+    file(INSTALL FILES "${source_file}" DESTINATION "${target_directory}" FOLLOW_SYMLINK_CHAIN)
 
     # Our Qt and Python frameworks aren't in the correct bundle format to be codesigned.
     if("${target_filename}" MATCHES "(Qt[^.]+)\\.[Ff]ramework")
-        fixup_qt_framework(${CMAKE_MATCH_1} "${target_directory}/${target_filename}")
+        fixup_qt_framework(${CMAKE_MATCH_1} "$ENV{DESTDIR}${CMAKE_INSTALL_PREFIX}/${target_directory}/${target_filename}")
```

Comment:
Should also strip the Qt libraries on Mac as well to reduce the size?

Reply:
stripping in mac is a bit different so I prefer leaving that for when we do the "installer" for mac
I just updated this call because I will likely forget
I added that stripping call in Linux because it didnt fit the deb max size.  
I would like to just strip directly in 3rdParty so we dont have to do it here and its also smaller 3rdparty packages. For our targets, `CPACK_STRIP_FILES` does the trick.